token/common.tact

```diff
@@ -1,5 +1,7 @@
 // WARNING: it may contains logic errors.
 
+let JettonAmount = newtype(Coins);
+
 struct WalletStateInit {
```

Comment:
is WalletStateInit still relevant in this code?

Reply:
Fixed in [fe0164c](https://github.com/tact-lang/tact-bikeshed/pull/5/commits/fe0164c40cc0c43eadd7ac984fd59c6ed6c0cdd9)